src/01_recoding.R

```diff
@@ -1,10 +1,6 @@
 library(dplyr)
-rec_2018 <- filter(
-  data_2018,
+data_pg <- filter(
+  data,
   country_name %in% ("Switzerland"), 
```

Comment:
@Federica-Giua se vuoi selezionare solo i casi svizzeri usa `=="Switzerland"`, `%in%` si usa quando hai un insieme di nomi ma qui ne avete solo uno.